README.md

```diff
@@ -242,6 +242,26 @@ Symbiote\QueuedJobs\Jobs\CleanupJob:
     - Complete
 ```
 
+## Health Checking
+
+Jobs track their execution in steps - as the job runs it increments the "steps" that have been run. Periodically jobs
+are checked to ensure they are healthy. This asserts the count of steps on a job is always increasing between health
+checks. By default health checks are performed when a worker picks starts running a queue.
+
+In a multi-worker environment this can cause issues when health checks are performed too frequently. You can disable the
+automatic health check with the following configuration:
+
+```yaml
+Symbiote\QueuedJobs\Services\QueuedJobService:
+  disable_health_check: true
```

Comment:
I'd prefer this to be the opposite i.e. `health_check: false` where it's `true` by default

Reply:
I felt like health checks are a given and you have to specifically disable them, but I don't really feel strongly.

Reply:
I also think about it that way @ScopeyNZ, and also do not feel strongly either way.